try{
    console.log(ace); //error occurs here
}
catch{
    console.log('error occured');
}
console.log('hello world'); // this will still execute
